internal/cmd/api-key/command_store.go

```diff
@@ -112,7 +112,7 @@ func (c *command) store(cmd *cobra.Command, args []string) error {
 		return errors.CatchApiKeyForbiddenAccessError(err, getOperation, httpResp)
 	}
 
-	apiKeyIsValidForTargetCluster := (cluster.ID == apiKey.Spec.Resource.Id)
+	apiKeyIsValidForTargetCluster := apiKey.Spec.Resource != nil && cluster.ID == apiKey.Spec.Resource.Id
```

Comment:
Can we use panic-safe getter functions here? (or does the SDK not allow it?)
```suggestion
	apiKeyIsValidForTargetCluster := cluster.ID == apiKey.GetSpec().GetResource().GetId()
```

Reply:
The getters are pointer methods, so you could do `apiKey.Spec.Resource.GetID()` which will prevent the same panic as adding the nil check above, so I think I'll do that.

Reply:
(We're still not able to chain multiple methods though, right?)

Reply:
Unfortunately no, Go doesn't allow for taking the addresses of pointers in a non-roundabout way (i.e. `apiKey.(&GetSpec()).(&GetResource()).GetID().